crt_portal/cts_forms/migrations/0175_update_outreach_tooltips.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.17 on 2023-06-23 14:18
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cts_forms', '0174_bannermessage'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='report',
+            name='origination_utm_campaign',
+            field=models.ForeignKey(blank=True, help_text='The internal name of the outreach effort.', null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='reports', to='cts_forms.campaign'),
+        ),
+        migrations.AlterField(
+            model_name='report',
+            name='origination_utm_content',
+            field=models.CharField(blank=True, help_text='A name for the specific link that was used to file the report.', max_length=100, null=True),
```

Comment:
I agree this is confusing, but we can't do styling in this help text since it's used in a few places (including a the admin panel and a style-free tooltip), unfortunately.

I might defer to [wikipedia's wording](https://en.wikipedia.org/wiki/UTM_parameters#UTM_parameters) here, but with a less marketing-jargon "such as":

> Identifies what specifically was clicked to bring the user to the site, such as to distinguish between two different outreach links on the same webpage or document.